modules/ibm-z-ensure-high-disk-performance-hyperpav.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ibm-z-recommended-host-practices.adoc
+
+[id="ibm-z-ensure-high-disk-performance-hyperpav_{context}"]
+= Ensure high disk performance with HyperPAV on z/VM
+
+DASD and ECKD devices are commonly used disk types in IBM Z environments. In a typical {product-title} setup in z/VM environments, DASD disks are commonly used to support the local storage for the nodes. You can set up HyperPAV alias devices to provide more throughput and overall better I/O performance for the DASD disks that support the z/VM guests.  
+
+Using HyperPAV for the local storage devices leads to a significant performance benefit. However, you must be aware that there is a trade-off between throughput and CPU costs.
+
+== Use the Machine Config Operator (MCO) to activate HyperPAV aliases using z/VM full-pack minidisks
```

Comment:
I think we should rephrase, not to confuse the user that he/she/it would be able to activate HyperPAV by using the minidisks. 
Proposal:
"_Use the Machine Config Operator (MCO) to activate HyperPAV aliases in nodes using z/VM full-pack minidisks_"